uniffi_bindgen/src/bindings/python/templates/wrapper.py

```diff
@@ -75,9 +75,6 @@
     {%- for c in ci.callback_interface_definitions() %}
     "{{ c.name()|class_name }}",
     {%- endfor %}
-    {%- if ci.has_async_fns() %}
```

Comment:
I don't think we want this change - this function is documented as being exported by the Python bindings.

Reply:
It's still exported. `__all__` just determines what you get when you do `from library import *`. You can still get it using `import library; library.uniffi_set_event_loop` or `from library import uniffi_set_event_loop`.

Reply:
I understand that - but I'm not sure this change will meet the expectations of people reading our docs.

Reply:
(to be clear, I'm not saying this change is wrong, I'm just not sure it's correct. @badboy @bendk do either of you have thoughts)

Reply:
Added such a note.